### PR TITLE
fix memo auth incorrect warning

### PR DESCRIPTION
**Pull request by AloeareV**

### CODE REVIEW

File: zingolib/src/blaze/fetch_full_transaction.rs [hunk 1]
```diff
@@ -184,28 +184,36 @@ impl TransactionContext {
...
+                                if let Some(out_metadata) =
+                                    transaction.outgoing_metadata.iter_mut().find(|out_meta| {
+                                        outgoing_potential_receivers
+                                            .contains(&Some(out_meta.to_address.clone()))
+                                    })
+                                {
+                                    out_metadata.ua = Some(ua.encode(&self.config.chain));
+                                } else {
+                                    log::error!(
+                                        "Recieved memo indicating you sent to \
```
Comment (zancas) [inline zingolib/src/blaze/fetch_full_transaction.rs:210]: `i` before `e` ....except....

**Comment by zancas** [comment]
I intend to test this before merge.

**Comment by AloeareV** [comment]
Blocked on tests

**Comment by AloeareV** [comment]
This is encompassed in #308

**Pull request closed without merging**